user_comments/contrib/is_core/cores.py

```diff
@@ -52,3 +64,6 @@ class CommentISCoreMixin:
             )
         }),
     )
+
+    def get_update_only_comments(self, request, obj=None):
```

Comment:
I think this method is not needed. Apart from having quite different name than the attribute it reads, which is confusing, developer can either define static attribute or define `@property` if dynamic outcome is desired. The args you're passing can be both accessed via `self`.

Or rename this method to `get_can_update_only_comment`.

Reply:
Renamed, since this (the method) is what @matllubos wanted, if I understood him correctly.

Reply:
No I wanted get_can_update_only_comment maybe I say you it with typo this is ok